students/Adil/Class_assignments/grid.py

```diff
@@ -0,0 +1,16 @@
+# Create Grid
+
+colCell = '+ - - - - + - - - -+'
+rowCell = '|         +        |'
+
+print(colCell)
+print(rowCell)
+print(rowCell)
+print(rowCell)
+print(rowCell)
+print(colCell)
+print(rowCell)
+print(rowCell)
+print(rowCell)
+print(rowCell)
+print(colCell)
```

Comment:
Hi Adil-

Check out [this link for some pointers](https://stackoverflow.com/questions/22584652/python-how-to-print-out-a-2d-list-that-is-formatted-into-a-grid) on how to solve this.